overlay/CMakeLists.txt

```diff
@@ -162,6 +162,7 @@ if(64_BIT AND MSVC)
 				"-DCMAKE_CXX_COMPILER=cl.exe"
 				"-DBUILD_OVERLAY_XCOMPILE=ON"
 				"-Dsymbols=${symbols}"
+				"-Dversion=${PROJECT_VERSION}"
```

Comment:
This does not match your changes of the variable's name

Reply:
As far as I can tell the `version` parameter does no longer exist. And setting it to the project's version doesn't seem to make a lot of sense anyways...
Or is this a variable exclusive to the overlay cmake scripts?

Reply:
https://github.com/mumble-voip/mumble/blob/fe086b41d83d6f488b5b91167c806a87094e32c0/overlay/CMakeLists.txt#L7-L18